packages/material-ui/src/Breadcrumbs/Breadcrumbs.js

```diff
@@ -2,41 +2,97 @@ import * as React from 'react';
 import { isFragment } from 'react-is';
 import PropTypes from 'prop-types';
 import clsx from 'clsx';
-import withStyles from '../styles/withStyles';
+import { deepmerge } from '@material-ui/utils';
+import { unstable_composeClasses as composeClasses } from '@material-ui/unstyled';
+import experimentalStyled from '../styles/experimentalStyled';
+import useThemeProps from '../styles/useThemeProps';
 import Typography from '../Typography';
 import BreadcrumbCollapsed from './BreadcrumbCollapsed';
+import breadcrumbsClasses, { getBreadcrumbsUtilityClass } from './breadcrumbsClasses';
 
-export const styles = {
-  /* Styles applied to the root element. */
-  root: {},
-  /* Styles applied to the ol element. */
-  ol: {
-    display: 'flex',
-    flexWrap: 'wrap',
-    alignItems: 'center',
-    padding: 0,
-    margin: 0,
-    listStyle: 'none',
+const overridesResolver = (props, styles) => {
+  return deepmerge(styles.root || {}, {
+    [`& .${breadcrumbsClasses.ol}`]: styles.ol,
+    [`& .${breadcrumbsClasses.li}`]: styles.li,
+    [`& .${breadcrumbsClasses.separator}`]: styles.separator,
+  });
+};
+
+const useUtilityClasses = (styleProps) => {
+  const { classes } = styleProps;
+
+  const slots = {
+    root: ['root'],
+    ol: ['ol'],
+    li: ['li'],
+    separator: ['separator'],
+  };
+
+  return composeClasses(slots, getBreadcrumbsUtilityClass, classes);
+};
+
+const BreadcrumbsRoot = experimentalStyled(
+  Typography,
+  {},
+  {
+    name: 'MuiBreadcrumbs',
+    slot: 'Root',
+    overridesResolver,
   },
-  /* Styles applied to the li element. */
-  li: {},
-  /* Styles applied to the separator element. */
-  separator: {
-    display: 'flex',
-    userSelect: 'none',
-    marginLeft: 8,
-    marginRight: 8,
+)(() => ({}));
+
+const BreadcrumbsOL = experimentalStyled(
+  'ol',
+  {},
+  {
+    name: 'MuiBreadcrumbs',
+    slot: 'OL',
```

Comment:
```suggestion
    slot: 'Ol',
```

It feels awkward to have elements as slots :\ @oliviertassinari what do you think of giving these slots semantic names in the future?

Reply:
I belive ol and li are acronyms for a semantic name order list, list item. It seems fine :)

Reply:
We could call the slot `List`, perhaps?

Reply:
In the end, the name + class name will be concatenated with the MuiBreadcrums in the beginning so it should be fine.. It's just that it's the first component I see we have that we use HTML element names as a reference to the slots :)